### PR TITLE
Trying implementing cat api

### PR DESCRIPTION
**Pull request by quangcito**
I changed:

- the API file since the JSON for cat API is an array
- I changed the data model for the cat API
- Tested on iphone 11 simulator

### CODE REVIEW

**Comment by quangcito** [comment]
@CarolineLaw Hey Spud! I have planned to change everything into Cat. I will slack you when I finish changing

**Comment by quangcito** [comment]
@CarolineLaw Hi Caroline, I have changed everything from dogs to cats + updated the Readme file. I am ready for a code review (hopefully by the end of this week). Thank you so much. I hope to hear from you soon.

File: RandomCats/RandomCatAPI.swift [hunk 1]
```diff
@@ -1,13 +1,14 @@
 //
-//  RandomDogAPI.swift
-//  RandomDogs
+//  RandomCatAPI.swift
+//  RandomCats
 //
-//  Created by Spud on 4/23/22.
+//  Created by Quang on 6/4/22.
```
Comment (SpudLaw) [inline RandomCats/RandomCatAPI.swift:5]: I think it's better to leave the "Created By" but add a "Edited By" :D you don't have to for this project, but in the future some developers might not take kindly to having their name erased haha no problemo for this project tho :]

File: RandomCatsUITests/RandomCatsUITests.swift [hunk 2]
```diff
@@ -1,13 +1,13 @@
 //
-//  RandomDogsUITests.swift
```
Comment (SpudLaw) [inline RandomCatsUITests/RandomCatsUITests.swift:2]: If you want to I can teach you how to write UI tests for this project, Apple's automated testing is really cool and fun to use.

File: RandomCatsTests/RandomCatsTests.swift [hunk 3]
```diff
@@ -1,14 +1,14 @@
 //
-//  RandomDogsTests.swift
```
Comment (SpudLaw) [inline RandomCatsTests/RandomCatsTests.swift:2]: we could come up with some unit tests for this project, probably on the API stuff mostly :]

**Review by SpudLaw — APPROVED** [review]
I left a couple of comments, but you don't have to address them :]

I haven't pulled down your code yet, but if you've tested this yourself and everything looks good to you I say merge this PR in :D

File: RandomCats/RandomCatAPI.swift (hunk 1 above)
Comment (quangcito) [reply RandomCats/RandomCatAPI.swift:5]: Haha thank you so much! Fully noted!

File: RandomCatsUITests/RandomCatsUITests.swift (hunk 2 above)
Comment (quangcito) [reply RandomCatsUITests/RandomCatsUITests.swift:2]: Yes, I would love to schedule with you a short session on writing UI tests.

File: RandomCatsTests/RandomCatsTests.swift (hunk 3 above)
Comment (quangcito) [reply RandomCatsTests/RandomCatsTests.swift:2]: Yes, that would be great. Let's meet sometime for a unit test and UI test lesson. Thank you so much!

**Pull request merged**